lib/kube/proxy/forwarder.go

```diff
@@ -1772,7 +1772,15 @@ func (f *Forwarder) portForward(authCtx *authContext, w http.ResponseWriter, req
 		return nil, trace.Wrap(err)
 	}
 
+	onForwardConnection := func(addr string, success bool) {
```

Comment:
I would switch these. 

for `onForwardConnection` you have access to which ports are being forwarded so you can emit real portforward events instead of doing nothing.

Reply:
This was the behavior before this PR, but that would be inconsistent with ssh forward and results in many events on each connection without any event when setting it up

Reply:
you can track which ports you already emit events for and skip them. It's important to note that auditing ports is important

Reply:
Done.